util/missingmacro.txt

```diff
@@ -88,6 +88,7 @@ LHASH_COMP_FN(3)
 LHASH_DOALL_ARG_FN(3)
 LHASH_OF(3)
 DEFINE_LHASH_OF(3)
+DEFINE_LHASH_OF_EX(3)
```

Comment:
It is not allowed to add things to missingmacro.txt meaning that DEFINE_LHASH_OF_EX should be properly documented.